src/Bootstrap.php

```diff
@@ -7,6 +7,15 @@
 
 namespace AchttienVijftien\Plugin\LastViewed;
 
+require plugin_dir_path( __FILE__ ) . 'Tracker.php';
```

Comment:
These aren''t needed.

Reply:
I thought so as well, but after activating the plugin, I got a fatal about not finding the classes.

Reply:
Double checked, the classes are added to the psr4 autoload file.

Reply:
I need this to work as if someone places these files in their plugin folder. That is what I did, and then I get fatals after activating. What file can we add to make everything autoload without someone having to do anything, but place the files?